### PR TITLE
refactor(DTFS2-4678): Remove commented out code

### PR DESCRIPTION
**Pull request by avaitonis**
- Removing commented out code
- Leaving // TODO: (Jira ref)   if it is present
- Updated Jira tickets to have code backup.

### CODE REVIEW

File: gef-ui/server/controllers/supporting-information/supporting-documents/index.test.js [hunk 1]
```diff
@@ -373,22 +373,6 @@ describe('controllers/supporting-documents', () => {
       });
     });
 
-    // it('returns error if error thrown when uploading', async () => {
```
Comment (abhi-markan) [inline gef-ui/server/controllers/supporting-information/supporting-documents/index.test.js:376]: Can you please add a TO-DO to write a test coverage around this please.

File: e2e-tests/trade-finance-manager/cypress/e2e/journeys/case/amendments/amendment-tasks-manual.spec.js [hunk 2]
```diff
@@ -443,18 +430,6 @@ context('Amendments tasks - manual amendment tasks', () => {
     pages.tasksPage.tasks.row(4, 4).status().contains('In progress');
     pages.tasksPage.tasks.row(4, 5).status().contains('Cannot start yet');
 
-    // TODO: add when add decision is dependent on tasks
```
Comment (abhi-markan) [inline e2e-tests/trade-finance-manager/cypress/e2e/journeys/case/amendments/amendment-tasks-manual.spec.js:446]: Can you please add a TO-DO to write a test coverage around this please.

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Few comments to address please.
Good work.

File: portal-api/api-tests/v1/gef/external-api.api-test.js [hunk 3]
```diff
@@ -13,7 +13,6 @@ xdescribe(baseUrl, () => {
   });
 
   beforeEach(async () => {
```
Comment (IgnatG) [inline portal-api/api-tests/v1/gef/external-api.api-test.js:15]: We can remove this completely if it's empty

File: e2e-tests/trade-finance-manager/cypress/e2e/journeys/case/amendments/amendment-tasks-manual.spec.js (hunk 2 above)
Comment (avaitonis) [reply e2e-tests/trade-finance-manager/cypress/e2e/journeys/case/amendments/amendment-tasks-manual.spec.js:446]: Done

File: gef-ui/server/controllers/supporting-information/supporting-documents/index.test.js (hunk 1 above)
Comment (avaitonis) [reply gef-ui/server/controllers/supporting-information/supporting-documents/index.test.js:376]: Done

File: portal-api/api-tests/v1/gef/external-api.api-test.js (hunk 3 above)
Comment (avaitonis) [reply portal-api/api-tests/v1/gef/external-api.api-test.js]: Done, "beforeEach" is removed

**Review by abhi-markan — APPROVED** [review]
LGTM

**Pull request merged**